core/src/main/java/com/exadel/kaliada/core/models/LikeModel.java

```diff
@@ -0,0 +1,30 @@
+package com.exadel.kaliada.core.models;
+
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.models.annotations.Default;
+import org.apache.sling.models.annotations.DefaultInjectionStrategy;
+import org.apache.sling.models.annotations.Model;
+import org.apache.sling.models.annotations.injectorspecific.ValueMapValue;
+
+@Model(adaptables = SlingHttpServletRequest.class,
+        resourceType = LikeModel.RESOURCE_TYPE,
```

Comment:
`resourceType` is not actually needed unless there's a real risk someone will try to adapt some `aemdemo/components/NOT_like` to the same model